### PR TITLE
Update row convolution in DS2 doc.

### LINKED ISSUE #2228: Add lookahead row convolution layer.
- Add lookahead row convolution layer, for both cpu and gpu version.
- Details please find in [DS2 paper](https://arxiv.org/abs/1512.02595).
- Please add a design-doc [here](https://github.com/xinghai-sun/Paddle/tree/ds2_design_doc/doc/design/speech#row-convolution) first.

### PR DESCRIPTION
**Pull request by qingqing01**
Fix #2228

### CODE REVIEW

File: doc/design/speech/README.MD [hunk 1]
```diff
@@ -136,7 +136,23 @@ CTC-Beam search                         | Not supported yet.
...
 ### Row Convolution
 
-TODO by Assignees
+- Difference with sequence convolution
+
+  The 1D sequence convolution (`paddle.layer.sequence_conv`) was supported, but this row convolution is different from it as shown in figure 2 and 3. In the figures, assumed the context length is 3 and the hidden size of each time step is `d`. The weight dimension of sequence convolution is `3d x d`. And weight dimension of row convolution is `3 x d`. The weights are shared among all time steps.
+
+<div align="center">
+<img src="image/seq_conv.png" width=350><br/>
+Figure 1. Connections of Sequence Convolution.
```
Comment (lcy-seso) [inline doc/design/speech/README.MD]: 1. figure 1 --> figure 2
2. the title : Connections of Sequence Convolution  --> sequence convolution

File: doc/design/speech/README.MD [hunk 2]
```diff
@@ -136,7 +136,23 @@ CTC-Beam search                         | Not supported yet.
...
+</div>
+
+<div align="center">
+<img src="image/row_conv.png" width=350><br/>
+Figure 1. Connections of Row Convolution.
```
Comment (lcy-seso) [inline doc/design/speech/README.MD]: 1. figure1 --> figure 3
2. the title : Connections of Row Convolution -->  Row Convolution

File: doc/design/speech/README.MD [hunk 3]
```diff
@@ -136,7 +136,23 @@ CTC-Beam search                         | Not supported yet.
 
 ### Row Convolution
 
-TODO by Assignees
+- Difference with sequence convolution
+
+  The 1D sequence convolution (`paddle.layer.sequence_conv`) was supported, but this row convolution is different from it as shown in figure 2 and 3. In the figures, assumed the context length is 3 and the hidden size of each time step is `d`. The weight dimension of sequence convolution is `3d x d`. And weight dimension of row convolution is `3 x d`. The weights are shared among all time steps.
```
Comment (lcy-seso) [inline doc/design/speech/README.MD]: the first sentence:   row convolution(`paddle.layer.row_conv`) is performed upon 1D sequence input, but it is different from 1D sequence convolution (`paddle.layer.sequence_conv`). The difference is shown in figure 2 and figure 3.

File: doc/design/speech/README.MD [hunk 4]
```diff
@@ -136,7 +136,23 @@ CTC-Beam search                         | Not supported yet.
...
+</div>
+
+<div align="center">
+<img src="image/row_conv.png" width=350><br/>
+Figure 1. Connections of Row Convolution.
+</div>
+
+- How to deal with boundary.
+
+  There are usually two modes for convolution, namely `same` and `valid` mode. The `same` mode means the output has the same size with input. The `valid` mode means the output are only computed without zero-padded edges. We only support the `same` mode for row convolution.
```
Comment (lcy-seso) [inline doc/design/speech/README.MD]: I think the explanation is a little simple.
- how the `same` mode make the output size is equal to the input size?
- and I think `size` here is a little confusing, does it mean the sequence length or dimension of the output vectors?

Comment (qingqing01) [inline doc/design/speech/README.MD]: Here, `size` = `sequence_length`  * `hidden_dim`, that is the number of input feature and output feature.  In fact, it mainly depends on sequence length.  In addition, the `same` and `valid` is frequently used in other platforms, such TensorFlow, Torch, MxNet. I'll explain it more in the doc.

Thanks for your comment. @lcy-seso

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献文档。由于文档已迁移至[FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo，因此关闭您的PR，欢迎您向[FluidDoc](https://github.com/PaddlePaddle/FluidDoc) Repo贡献文档。
Thanks for contributing to PaddlePaddle! Since documents have been moved to [FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo, we close this PR. Welcome to contribute to [FluidDoc](https://github.com/PaddlePaddle/FluidDoc) repo.

**Pull request closed without merging**